x-pack/test/functional/apps/ml/anomaly_detection_jobs/job_list.ts

```diff
@@ -96,51 +76,43 @@ export default function ({ getService }: FtrProviderContext) {
 
     after(async () => {
       await ml.api.closeAnomalyDetectionJob(jobId);
-      await ml.api.cleanMlIndices();
       await ml.api.deleteAnomalyDetectionJobES(jobId);
+      await ml.api.cleanMlIndices();
       await ml.testResources.deleteDataViewByTitle('ft_farequote');
     });
 
     it('expanded row with connected calendar shows link to calendar', async () => {
-      await ml.testExecution.logTestStep('calendar present for connected job');
-      await ml.expandedJobDetails.assertCalendarPresent(jobId, calendarId);
+      await ml.expandedJobDetails.assertJobRowCalendars(jobId, [calendarId]);
     });
 
     it('expanded row with forecast should display open forecast button', async () => {
-      await ml.testExecution.logTestStep('open forecast link present');
       await ml.expandedJobDetails.assertForecastElements(jobId);
     });
 
     it('expanded row with annotations can be edited', async () => {
-      await ml.testExecution.logTestStep('can edit annotations from expanded row');
-      await ml.expandedJobDetails.assertAnnotationsEdit(jobId);
+      await ml.expandedJobDetails.editAnnotation(jobId, 'appex qa rox');
```

Comment:
While I definitely agree to this statement, I'd still rather keep our production code on a more neutral language :wink: